src/systems/isq/include/units/isq/mechanics.h

```diff
@@ -62,7 +61,7 @@ QUANTITY_SPEC(normal_stress, pressure, quantity_character::scalar);
 QUANTITY_SPEC(shear_stress, pressure, quantity_character::scalar);
 QUANTITY_SPEC(strain, dimensionless, quantity_character::tensor);
 QUANTITY_SPEC(relative_linear_strain, length / length);
-QUANTITY_SPEC(shear_strain, displacement / thickness);
+QUANTITY_SPEC(shear_strain, displacement / thickness);  // TODO scalar or vector?
```

Comment:
It's defined as a scalar.

Reply:
I do not see that it is explicitly defined as a scalar, and it being a quotient of displacement, which is a vector, made it a vector in the library, which was a bit surprising to find out in the unit test.

Reply:
Well, it's symbol is not bold. Neither is the symbol of the displacement in the formula. And if you read the description, it seems to be referring to the parallel component of the vector.